### PR TITLE
update for tensorflow 12

### PR DESCRIPTION
**Pull request by jrock08**

### CODE REVIEW

File: tests/test_tftools.py [hunk 1]
```diff
@@ -86,9 +86,9 @@ def test_batch_norm_2d():
     training = tf.placeholder(tf.bool, [])
     y = layers.batch_norm(x, training)
     sess = tf.Session()
-    sess.run(tf.initialize_all_variables())
-
     x_ = np.float32(np.random.randn(*input_shape))
+    
+    sess.run(tf.initialize_all_variables(), feed_dict={x:x_, training:True})
```
Comment (jrock08) [inline tests/test_tftools.py]: I have no idea why I have to feed these to initialize...

File: .travis.yml [hunk 2]
```diff
@@ -20,7 +20,7 @@ install:
   - conda create -q -n test-environment python=$TRAVIS_PYTHON_VERSION numpy pytest
   - source activate test-environment
   - easy_install --upgrade six
-  - pip install --upgrade https://storage.googleapis.com/tensorflow/linux/cpu/tensorflow-0.10.0-cp27-none-linux_x86_64.whl
+  - pip install --upgrade https://storage.googleapis.com/tensorflow/linux/cpu/tensorflow-0.11.0-cp27-none-linux_x86_64.whl
```
Comment (BigRedT) [inline .travis.yml]: why does it say 11? The latest release is 0.12.1

Comment (jrock08) [inline .travis.yml]: Oh, I  was going to do an 10->11 update then an 11->12 update in this one pull request, but when the 10->11 went funny, I wanted to figure out why before moving on.

File: aiutils/tftools/batch_normalizer.py [hunk 3]
```diff
@@ -1,5 +1,5 @@
 import tensorflow as tf
-from tensorflow.python import control_flow_ops
+from tensorflow.python.ops import control_flow_ops
```
Comment (BigRedT) [inline aiutils/tftools/batch_normalizer.py]: control_flow_ops isn't referenced anywhere in the code and can be deleted.

File: tests/test_tftools.py (hunk 1 above)
Comment (BigRedT) [inline tests/test_tftools.py]: https://www.tensorflow.org/api_docs/python/state_ops/exporting_and_importing_meta_graphs#initialize_all_variables

we should switch to `tf.global_variables_initializer`

Comment (BigRedT) [inline tests/test_tftools.py]: same for rest of initialize calls in this file

Comment (jrock08) [inline tests/test_tftools.py]: I'd prefer marking that as a bugfix/todo to keep upgrades to new tf versions as simple as possible.

File: .travis.yml (hunk 2 above)
Comment (BigRedT) [inline .travis.yml]: let's try 12.1 directly

Comment (jrock08) [inline .travis.yml]: Yeah, will push, for 11, we should accept #50 if it causes no issues.

File: tests/test_tftools.py (hunk 1 above)
Comment (BigRedT) [inline tests/test_tftools.py]: sure

File: requirements.txt [hunk 4]
```diff
@@ -4,3 +4,4 @@ pillow
 pytest
 tensorflow
 future
+scipy
```
Comment (BigRedT) [inline requirements.txt]: where are we using this?

**Pull request merged**